eosdis-nasa/earthdata-pub-api
# Edpub 1215 sample file upload questions

# Description

Adds file inputs for upload capability in forms and updates the verbiage for the upload questions

## Spec


See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1215

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Deploy the api, dashboard, and forms apps locally.
4. Create new Request for any DAAC
5. Verify that the DAR form shows the new working and file upload option in addition to the URL field.
